keycloak-iam/src/main/kotlin/com/valtimo/keycloak/security/config/RoleSecurityConfigurer.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.valtimo.keycloak.security.config
+
+import com.ritense.valtimo.contract.security.config.HttpSecurityConfigurer
+import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import com.ritense.valtimo.contract.authentication.AuthoritiesConstants.ADMIN
+import com.ritense.valtimo.contract.security.config.HttpConfigurerConfigurationException
+
+
+class RoleSecurityConfigurer: HttpSecurityConfigurer {
+
+    override fun configure(http: HttpSecurity) {
+        try {
+            http.authorizeRequests()
+                .antMatchers(HttpMethod.GET, "/api/v1/role").hasAnyAuthority(ADMIN)
```

Comment:
Should this only be available to admins?

Reply:
added USER

Reply:
@ivo-ritense shouldn't this be covered by PBAC? I thought we're not using ROLE_USER any more.